app/services/step_factory.rb

```diff
@@ -1,19 +1,81 @@
 class StepFactory
```

Comment:
Does this essentially hold the same information as the `Candidate` model constants but in a slightly different format? if something changes would we have to update both files?

Reply:
No, StepFactory is the model classes/ urls, Candidate is the required session values for the api, they are not necessarily dependant.